basis/html/templates/chloe/compiler/compiler.factor

```diff
@@ -149,9 +175,8 @@ ERROR: unknown-chloe-tag tag ;
 
 : compile-prologue ( xml -- )
     [
-        [ prolog>> [ write-xml ] [code-with] ]
-        [ before>> compile-chunk ]
-        bi
+        "<!DOCTYPE html>\n" [write]
```

Comment:
XHTML wasn't ever terribly well supported, and actually has a negative impact in some browsers (e.g. it'll push IE into one of its quirks modes), so I *think* that fully drinking the HTML5 Koolaid is okay at this point. Certainly XHTML is no longer being developed and HTML5 is.

If there are reasons I'm not aware of why we want to keep *Chloe* XHTML generation alive, let me know; the other option I debated was revving Chloe to 1.1 and only doing the `DOCTYPE` swizzle if that's set. It's not much more code; I just didn't see any value.